### PR TITLE
RHCLOUD-21170: Add cyndi ConfigMap when kafka provider is managed-ephem

### PR DESCRIPTION
**Pull request by bsquizz**
We need to tweak the connector topic, "deadletterqueue" topic, and topic replication factor for cyndi to work properly in managed-ephem kafka environments.

Related to: https://github.com/RedHatInsights/cyndi-operator/pull/72

### CODE REVIEW

File: controllers/cloud.redhat.com/providers/kafka/managed-ephem.go [hunk 1]
```diff
@@ -326,16 +326,29 @@ func (mep *managedEphemProvider) configureBrokers() error {
...
 
-func (mep *managedEphemProvider) createCyndiPipeline(app *crd.ClowdApp) error {
+func (mep *managedEphemProvider) configCyndi(app *crd.ClowdApp) error {
 	if !app.Spec.Cyndi.Enabled {
 		return nil
 	}
-	return createCyndiPipeline(mep, app, getConnectNamespace(mep.Env), getConnectClusterName(mep.Env))
+
+	// TODO: as a followup, make these topic names configurable
+	eventsTopic := fmt.Sprintf("%s-platform.inventory.events", mep.Env.Name)
```
Comment (psav) [inline controllers/cloud.redhat.com/providers/kafka/managed-ephem.go:335]: Honestly, I'd move these two topic name strings into the other function, then the prototype becomes a lot simpler.

**Review by psav — CHANGES_REQUESTED** [review]

**Comment by psav** [comment]
Hey - a little confused in this one, I know the change to cyndi will affect both types of operator. The test seems to be called `managed-kafka` but the mode is set to `operator`. If it is an operator mode test, it shouldn't be called `managed kafka` and if it's managed kafka, it shouldn't have mode of `operator`.

**Comment by bsquizz** [comment]
Yep the confusion is warranted -- the test should be using 'managed-ephem' mode, will change it.

File: controllers/cloud.redhat.com/providers/kafka/managed-ephem.go (hunk 1 above)
Comment (bsquizz) [reply controllers/cloud.redhat.com/providers/kafka/managed-ephem.go]: Moved

**Comment by bsquizz** [comment]
Gave up on writing a kuttl test -- the dependency on contacting managed kafka is too ingrained so at the moment, too much communication w/ managed kafka occurs before we get to the cyndi codepath.

**Review by psav — APPROVED** [review]

**Pull request merged**